youtube_dl/__init__.py

```diff
@@ -169,7 +169,7 @@ def _real_main(argv=None):
         if not opts.audioquality.isdigit():
             parser.error('invalid audio quality specified')
     if opts.recodevideo is not None:
-        if opts.recodevideo not in ['mp4', 'flv', 'webm', 'ogg', 'mkv']:
+        if opts.recodevideo not in ['mp4', 'flv', 'webm', 'ogg', 'mkv', 'xvid']:
```

Comment:
@aurium I am confused, what is your argument? That AVI should not be the default container for XviD video? If so I agree with @dstftw, you need to make a case for that, because while it is not the only container that supports XviD, it is certainly the defacto one.